google/cloud/storage/client.h

```diff
@@ -3492,6 +3493,10 @@ class Client {
   StatusOr<PolicyDocumentV4Result> SignPolicyDocumentV4(
       internal::PolicyDocumentV4Request request);
 
+  std::string Endpoint() const;
+
+  std::string Host() const;
```

Comment:
"Host" seems a bit too generic.  Consider: `EndpointHost`?  Or maybe `EndpointAuthority()` (an obscure, but AFAIK correct, name for the `hostname:port` part of the endpoint URL?)

In addition, consider a comment.